### ISSUE TITLE
Preserve Trac comment numbers

### ISSUE THREAD

**Issue by mkoeppe** [sagemath/trac-to-github#7]
Currently [the script](https://github.com/sagemath/trac-to-github/blob/master/migrate.py#L311) converts Trac comments to GH Issue comments with a header like this:
```
Comment by @mkoeppe created at 2022-07-04 20:14:00
```
(https://github.com/sagemath/trac_to_gh/issues/26#issuecomment-1245532523)

It would be good to preserve the comment number, for the benefit of external links pointing to specific comments.
For example, we could just include the full original URL:
```
Comment by @mkoeppe created at 2022-07-04 20:14:00 (original: https://trac.sagemath.org/ticket/34110#comment:4)
```

Likewise, [the script](https://github.com/sagemath/trac-to-github/blob/master/migrate.py#L502) should show the original Trac URL of the ticket.

**Comment by kwankyu** [comment]
If we create links to trac in the converted issues and comments, then those links will be broken when trac is stopped someday. Is this okay?

**Comment by dimpase** [comment]
the idea is to either rewrite these links, or keep a readonly copy of trac around

**Comment by kwankyu** [comment]
Why create new links to trac if we rewrite them after migration?

I think it makes sense to embed the original comment number in the migrated comment but embedding a link to the original comment does not (if trac goes down someday)

**Comment by mkoeppe** [comment]
Separate from the question whether we want to display the trac links:

I think we need to keep a mapping (ticket_number, comment_number) -> issue_comment.url so that we can rewrite Trac links to specific comments

**Comment by kwankyu** [comment]
Yes.

**Comment by kwankyu** [comment]
After ticket-to-issue migration, would the issues (and comments therein) be editable? If yes, who will have the write permission? The original author?

**Comment by mkoeppe** [comment]
Yes, the original author of the issue or comment -- if we have a github account for them on file.

**Comment by mkoeppe** [comment]
[comment 4711](#comment%3A4711): Perhaps we can use markdown anchors in the issue comments

**Comment by mkoeppe** [comment]
<a id='comment:4711'></a>

**Comment by mkoeppe** [comment]
I'm writing these anchors out in 23a993f. 

Help welcome with rewriting links in "Replying to [comment:18 jdemeyer]:" etc. to markdown

**Comment by mkoeppe** [comment]
This is working well, thank you!

**Comment by mkoeppe** [comment]
Fixed by #45
